### ISSUE TITLE
Use a local (server) folder instead of iCloud

### ISSUE THREAD

**Issue by samplue** [latenitefilms/RecallToolbox#2]
An option to use a local (shared) folder so that you can add items without the need to upload them to iCloud, but instead using your own local server for example

**Comment by latenitefilms** [comment]
This is slightly more tricky, and probably not something I can easily add to v1, but we'll see.

**Comment by samplue** [comment]
If it makes it any easier: Support for using a local folder instead of a shared folder. We use Syncthing to sync projects and folders in our office.

**Comment by latenitefilms** [comment]
Not really... the issue is that I use Core Data for Recall Toolbox, so Apple pretty much handles all the heavy lifting. It's a bit of an engineering challenge to pull data from multiple sources, but let me investigate. Definitely possible, just hard to build I think. We'll see.
